src/Core/EspWindow.cc

```diff
@@ -156,12 +159,60 @@ namespace esp
                              });
   }
 
-  void EspWindow::create_window_surface(VkInstance instance, VkSurfaceKHR* surface)
+  void EspWindow::handle_keys_presses()
+  {
+    if (glfwGetKey(m_window, GLFW_KEY_W) == GLFW_PRESS)
```

Comment:
This approach will yield a giant wall of ifs when we decide to handle more keys. Instead we could define an array of keys to handle and check for them in a loop.